spec/models/target_spec.rb

```diff
@@ -8,6 +8,22 @@
     it { should validate_presence_of(:latitude) }
     it { should validate_presence_of(:longitude) }
     it { should validate_presence_of(:user) }
+
+    context 'when a user has 10 targets and tries to create another one' do
+      let!(:user_1) { create :user }
+      it 'should not be able to do so' do
```

Comment:
Don't start tests with `should`. 
The context can be `when the user has already reached the target count limit`
It`does not allow to create another targer`.

Reply:
Actually this test does not sound necessary to me. The test below is correct and it implies that since the target is not valid then it will not get saved to the db. Consider putting the test below inside of `describe 'validations'` and dropping this one.